Standards/scs-0218-v1-container-registry-for-scs-standard-implementation.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Requirements for container registry
-type: Decision Record
+title: Container registry for SCS standard implementation
+type: Decision Record 
```

Comment:
```suggestion
type: Decision Record
```

Reply:
Changed in https://github.com/SovereignCloudStack/standards/pull/458/commits/e73ed463c4d03e128dcfa7f1437bfa7131de3f93